tests/terraform/graph/checks_infra/attribute_solvers/number_of_words_greater_than_or_equal_solver/test_solver.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from tests.terraform.graph.checks_infra.test_base import TestBaseSolver
+
+TEST_DIRNAME = os.path.dirname(os.path.realpath(__file__))
+
+
+class TestNumberOfNotWordsGreaterThanOrEqual(TestBaseSolver):
```

Comment:
all the test classes have a `Not` inside on purpose or copy-paste? 😄

Reply:
yes (haha) will fix :)